subsys/net/lib/openthread/platform/radio.c

```diff
@@ -279,7 +274,8 @@ static void openthread_handle_received_frame(otInstance *instance,
 {
 	otRadioFrame recv_frame;
 
-	recv_frame.mPsdu = net_buf_frag_last(pkt->buffer)->data;
+	recv_frame.mPsdu = net_buf_frag_last(pkt->buffer) ?
```

Comment:
Again, I'm not sure if that's even possible to get a NULL pointer here, unless we were provided with a buggy `net_pkt`, but if we need want to protect against it, I think we should verify the pointer and exit early instead of processing a frame with no payload (which makes little sense IMO).